src/datetime/tests.rs

```diff
@@ -1186,7 +1186,10 @@ fn test_subsecond_part() {
 }
 
 #[test]
-#[cfg(feature = "std")]
+#[cfg(all(feature = "std", not(target = "wasm32-wasi")))]
+// Some targets, such as `wasm-wasi` have a problematic definition of `SystemTime`, such as an
+// `i32` (year 2035 problem), or an `u64` (no values before `UNIX-EPOCH`).
+// See https://github.com/rust-lang/rust/issues/44394.
```

Comment:
Nit: I'd like the comments to go before the attributes, keeping the attributes right next to the code.